src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Product/Tabs/basic.html.twig

```diff
@@ -28,6 +28,29 @@
 <div role="tabpanel" class="form-contenttab tab-pane container-fluid active" id="basic-tab">
   <div class="row">
     <div class="col-md-9 left-column">
+      <div 
+        id="product-images-container"
+        data-translations="{{
+          {
+            'window.selectAll': 'Select all'|trans({}, 'Admin.Catalog.Feature'),
+            'window.unselectAll': 'Unselect all'|trans({}, 'Admin.Catalog.Feature'),
+            'window.selectedFiles': 'selected file(s)'|trans({}, 'Admin.Catalog.Feature'),
```

Comment:
I think the wording should include a placeholder ti inject the number Because in some languages the number won't be at the beginning of the sentence so it should belong to the wording